controllers/subscriptions.go

```diff
@@ -44,10 +44,10 @@ var getSubscriptions = func(orgID string) types.SubscriptionsResponse {
 	}
 
 	resp, err := getClient().Get(config.GetConfig().Options.GetString(config.Keys.SubsHost) +
-		"/svcrest/subscription/v5/search/criteria" +
+		"/svcrest/subscription/v5/searchnested/criteria" +
 		";web_customer_id=" + orgID +
-		";sku=SVC3124" +
-		";status=active")
+		";sku=SVC3851,SVC3852,SVCSER0566,SVCSER0567,SVC3124,RH00068" +
+		";status=active/options;products=ALL/product.sku")
```

Comment:
Just found out that *RH00068* is a parent sku. At this point in time, entitlements only checks for child skus, since we are used status=active all this time. Parent skus never have a status.